lib/internal/quic/quic.js

```diff
@@ -0,0 +1,2481 @@
+'use strict';
+
+const {
+  ArrayBuffer,
+  ArrayIsArray,
+  ArrayPrototypePush,
+  BigUint64Array,
+  DataView,
+  DataViewPrototypeGetBigInt64,
```

Comment:
We can remove all primordials with prototypes from this file, just like http and http2.

Reply:
I think for now I'll leave them in. I can go back in a future iteration to change this.

Reply:
Or I can help? :-)

Reply:
Can do that also but I want to focus mostly on getting this ready to land first. Once we get a good CI result on this if you'd like to push a commit that removes the primordials then +1